components/resources/permissions_strings.grdp

```diff
@@ -21,4 +21,10 @@
   <message name="IDS_PERMISSIONS_BUBBLE_SITE_PERMISSION_LINK" desc="Site permission link in the footnote description.">
     site permission
   </message>
+  <message name="IDS_OPEN_AI_CHAT_PERMISSION_FRAGMENT" desc="Label for Leo AI chat permission prompt.">
+    Allow to open Leo in Brave?
```

Comment:
also, can we update the icon in the permission prompt? The same icon you use in the WebUI should be available

Reply:
Will open a follow-up issue for this.